src/stylesheets/elements/_headings.scss

```diff
@@ -196,7 +196,8 @@ article + aside > h2 {
 /*--- Card styles ---*/
 
 .card,
-[class*="card--"] {
+.card--large,
+.card--small {
```

Comment:
Does this effect card__details in a negative way?

Reply:
Hi, I did this to stop any headings from being styled in the wrapper named `.card--extended`. This way any headings inside any other card including cards inside card--extended will display the specific format specified in this file.